python/rules_engine.py

```diff
@@ -247,7 +248,9 @@ def v602(self):
 		"""An invalid Calendar Quarter was reported. 1) Calendar Quarter must equal 4, and cannot be left blank."""
 		field = "calendar_quarter"
 		edit_name = "v602"
-		fail_df = self.ts_df[self.ts_df.calendar_quarter!=str(4)]
+		fail_df = self.ts_df.copy()
+		fail_df.calendar_quarter = fail_df.calendar_quarter.apply(lambda x: int(x))
```

Comment:
All the others are string comparisons, why does this one need int?

Reply:
The data map has a configuration for calendar quarter stored as an integer. This changes is to have the validation use all variables with similar types.